bluesky/plans.py

```diff
@@ -1631,10 +1629,19 @@ def list_scan(detectors, motor, steps, *, per_step=None, md=None):
         md,
         {'detectors': [det.name for det in detectors],
          'motors': [motor.name],
+         'num_steps': len(steps),
          'plan_args': {'detectors': list(map(repr, detectors)),
                        'motor': repr(motor), 'steps': steps,
                        'per_step': repr(per_step)},
-                       'plan_name': 'list_scan'})
+         'plan_name': 'list_scan',
+
+         # TODO what to do in this case?
+         'plan_pattern': 'list',
```

Comment:
This is the only place where the kwarg-only `plan_pattern_args` fails. Is it ok to special-case this? What's your preference?